index.html

```diff
@@ -61,10 +61,8 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
           <div>
             <label><input type="radio" id="check3" class="checkbox" name="attribute" value="Not sure">Not sure</label>
           </div>
-
         </div>
         <div class="formulary">
-
         </div>
         <div class="formulary">
 

```

Comment:
Please Apply best practices in HTML code, It will help the interactions with the browser and bring to the other programmers a clean code to read